src/main/java/com/spring/boot/service/CsvParserService.java

```diff
@@ -0,0 +1,7 @@
+package com.spring.boot.service;
+
+import java.util.List;
+
+public interface CsvParserService<T> {
+    List<T> parcFromCsvToObjects(String path);
```

Comment:
```suggestion
    List<T> parseCsvFile(String path);
```
Let's give short and meaningful names for methods

Reply:
Done